src/app/users/page.tsx

```diff
@@ -0,0 +1,26 @@
+
+import { fetchUsers } from '../services/userService';
+
+
+
+
+export default async function UsersPage() {
```

Comment:
We can't use `async` like this.
```suggestion
export default function UsersPage() {
```